### PR TITLE
Enable regression test for add loop benchmarks

### PR DESCRIPTION
**Pull request by laithsakka**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #136573

The red dotted line is 1.5

Screenshot 2024-09-24 at 11 50 41 AM

expected taken from the average.
Screenshot 2024-09-24 at 2 33 27 PM

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames @rec

### CODE REVIEW

**Comment by laithsakka** [comment]
failure not related.

File: benchmarks/dynamo/pr_time_benchmarks/expected_results.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+add_loop_eager,                compile_time_instruction_count, 2834456320,  0.015
+add_loop_eager_dynamic,        compile_time_instruction_count, 5528896630,  0.015
+add_loop_inductor,             compile_time_instruction_count, 24146845503, 0.015
+add_loop_inductor_dynamic_gpu, compile_time_instruction_count, 39411706509, 0.015
+add_loop_inductor_gpu,         compile_time_instruction_count, 22171041650, 0.015
```
Comment (ezyang) [inline benchmarks/dynamo/pr_time_benchmarks/expected_results.csv:5]: So how do I update these when they change?

**Review by ezyang — APPROVED** [review]

**Comment by laithsakka** [comment]
@pytrochbot merge

**Comment by laithsakka** [comment]
set thredhold for add_loop_eager_dynamic and add_loop_inductor_dynamic_gpu to 2.5 there was a recent interesting flaky point with

**Comment by laithsakka** [comment]
rebase

**Pull request closed without merging**